tests/pass/shims/env/getpid_with_isolation.rs

```diff
@@ -0,0 +1,7 @@
+//@only-target-linux
```

Comment:
This test only uses std so it should work on all targets, no?

Reply:
I was about to write a comment explaining this (I changed the test to only skip windows now):

Additionally the `getpid_with_isolation` test was failing for window targets since my PR only changed the `getpid` function, but windows. So I skip the test for windows. However it would also make sense to make similar changes to support `GetCurrentProcessId` in isolation mode, but I don't know if that is wanted. Let me know.

Reply:
If it's a small change it would be rather nice to have this test run on all our tested targets :)

Reply:
done!